code/controllers/subsystem/parallax.dm

```diff
@@ -46,9 +59,16 @@ SUBSYSTEM_DEF(parallax)
 		currentrun = queued
 		queued = temp
 		current_run_pointer = 1
+
 		//Check client count
-		throttle_ghosts = throttle_ghost_pop && length(GLOB.clients) >= throttle_ghost_pop
-		throttle_all = throttle_all_pop && length(GLOB.clients) >= throttle_all_pop
+		// TODO: should be changed based on how expensive the new parallax rendering is
+		// I just can't think a detailed way to reduce parallax lag when the new system has no data for how it is effective than before
+		wait = initial(wait)
+		if(throttle_ghost_pop && length(GLOB.clients) >= throttle_ghost_pop)
+			wait++
+		if(throttle_all_pop && length(GLOB.clients) >= throttle_all_pop)
+			wait *= 2
```

Comment:
The throttling was probably the most important part of the entire subsystem, the lag mitigation from the enumeration is great, but at high-pops a good majority of the time was spent processing the parallax for ghosts. Ghosts are extremely problematic at high pop because they are extremely fast and extremely plentiful. They cause everyones parallax to get lag mitigated. You really, really, should not delete the freezing of parallax layers for ghosts at high-pop since it will just bring back parallax's running slowly for humans at higher population limits.